crm_compassion/models/interaction_resume.py

```diff
@@ -58,106 +58,117 @@ def populate_resume(self, partner_id):
+                            ON pcj.config_id = c.id
+                            AND c.name != 'Default communication'
+                        WHERE pcj.state = 'done'
+                        AND pcj.send_mode = 'physical'
+                        AND (p.contact_id in ({}) OR p.id in ({}))
+            -- phonecalls
+                    UNION (
+                      SELECT
+                        'Phone' as communication_type,
+                        crmpc.date as communication_date,
+                        COALESCE(p.contact_id, p.id) AS partner_id,
+                        NULL AS email,
+                        crmpc.name as subject,
+                        crmpc.name as body,
+                        CASE crmpc.direction
+                            WHEN 'inbound' THEN 'in' ELSE 'out'
+                            END
+                        AS direction,
+                        crmpc.id as phone_id,
+                        0 as email_id,
+                        0 as message_id,
+                        0 as paper_id,
+                        NULL as tracking_status
+                        FROM "crm_phonecall" as crmpc
+                        JOIN res_partner p ON crmpc.partner_id = p.id
+                        WHERE (p.contact_id in ({}) OR p.id in ({}))
+                        )
+            -- outgoing e-mails
+                    UNION (
+                      SELECT
+                        'Email' as communication_type,
+                        m.date as communication_date,
+                        COALESCE(p.contact_id, p.id) AS partner_id,
+                        mt.recipient_address as email,
+                        COALESCE(config.name, m.subject) as subject,
+                        REGEXP_REPLACE(m.body, '<img[^>]*>', '') AS body,
+                        'out' AS direction,
+                        0 as phone_id,
+                        mail.id as email_id,
+                        0 as message_id,
+                        job.id as paper_id,
+                        COALESCE(mt.state, 'error') as tracking_status
+                        FROM "mail_mail" as mail
+                        JOIN mail_message m ON mail.mail_message_id = m.id
+                        JOIN mail_mail_res_partner_rel rel
+                        ON rel.mail_mail_id = mail.id
+                        FULL OUTER JOIN mail_tracking_email mt ON mail.id = mt.mail_id
+                        JOIN res_partner p ON rel.res_partner_id = p.id
+                        FULL OUTER JOIN partner_communication_job job
+                            ON job.email_id = mail.id
+                        FULL OUTER JOIN partner_communication_config config
+                            ON job.config_id = config.id
+                            AND config.name != 'Default communication'
+                        WHERE mail.state = ANY (ARRAY ['sent', 'received'])
+                        AND (p.contact_id in ({}) OR p.id in ({}))
```

Comment:
These parameters will be replaced by `partners_with_same_email_ids`
```suggestion
                        AND (p.contact_id = ANY(%s) OR p.id = ANY(%s))
```